lsof.adoc

```diff
@@ -0,0 +1,22 @@
+=== sysutils/lsof major upgrade
+
+Links
+link:https://github.com/lsof-org/lsof[lsof project repo] URL: link:https://github.com/lsof-org/lsof[https://github.com/lsof-org/lsof] +
+
+Contact: Larry Rosenman <ler@FreeBSD.org>
+
+sysutils/lsof in ports has had a major upgrade to no longer go looking 
+in /dev/kmem for data and using userspace API's.  This has taken a long
```

Comment:
```suggestion
in /dev/kmem for data and using the userland API.  This has taken a long
```
Single API, comprising a bunch of system calls, device data formats, etc. Or if you see it as a file IO API, a process control API, a resource allocation API, etc., that is one API per functional group, I'd still remove the ': APIs.